Cargo.toml

```diff
@@ -6,3 +6,8 @@ version = "0.3.1"
 edition = "2021"
 license = "Apache-2.0"
 
+[features]
+sanitise = ["sanitize-filename-reader-friendly"]
+
+[dependencies]
+sanitize-filename-reader-friendly = { version = "2.2.1", optional = true }
```

Comment:
Or should we use the other crate?

Reply:
I think the dependency ought to be removed entirely.